kratos/utilities/element_size_calculator.h

```diff
@@ -116,6 +138,67 @@ class KRATOS_API(KRATOS_CORE) ElementSizeCalculator {
 
     ///@}
 
+private:
+    ///@name Private Operations
+    ///@{
+
+    static inline double HsqDerivative2D(
+        const double Vx,
+        const double VxDerivative,
+        const double Nx,
+        const double NxDerivative,
+        const double Vy,
+        const double VyDerivative,
+        const double Ny,
+        const double NyDerivative)
+    {
+        return 2 * (Vx * Nx + Vy * Ny) *
+                   (VxDerivative * Nx + Vx * NxDerivative + VyDerivative * Ny +
+                    Vy * NyDerivative) / (Nx * Nx + Ny * Ny) -
+               std::pow((Vx * Nx + Vy * Ny) / (Nx * Nx + Ny * Ny), 2) *
+                   (2 * Nx * NxDerivative + 2 * Ny * NyDerivative);
+    }
```

Comment:
Why is this implemented in the header?

Reply:
I wanted to have those methods inlined. Thats the only reason. Is there any other way around it?